tests/testROChange.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+import subprocess
+import os
+import requests
+import json
+import sys
+import base64
+import time
+import traceback
+import urllib
+import logging
+import datetime
+from jwkest.jws import JWS
+from jwkest.jwk import RSAKey, import_rsa_key_from_file, load_jwks_from_url, import_rsa_key
+from jwkest.jwk import load_jwks
+from Crypto.PublicKey import RSA
+from WellKnownHandler import WellKnownHandler, TYPE_SCIM, TYPE_OIDC, KEY_SCIM_USER_ENDPOINT, KEY_OIDC_TOKEN_ENDPOINT, KEY_OIDC_REGISTRATION_ENDPOINT, KEY_OIDC_SUPPORTED_AUTH_METHODS_TOKEN_ENDPOINT, TYPE_UMA_V2, KEY_UMA_V2_PERMISSION_ENDPOINT
+from eoepca_uma import rpt, resource
+
+class ROChangeTest(unittest.TestCase):
```

Comment:
This is a really big test case. If we are focusing in unit tests, we should try to split functionality as much as possible (this might require splitting functions in single-purpose code)